internal/types/event.go

```diff
@@ -1,5 +1,10 @@
 package types
 
+type EventEnvelope struct {
+	Event Event
+	Key   []byte
```

Comment:
Will the `Key` always be the cluster id?

Reply:
The message Key will *always* be cluster id. The reason is that the topic we are sending to is split into N partitions (6), and the messages get sharded in function of the Key: a message with a given key will always end up in the same partition. Partitioning allow us to leverage consumer groups for faster processing, using cluster ID as key ensure we maintaing order among messages of the same cluster.

Reply:
Nice, thank you for the explanation!